frontend/src/container/OnboardingContainer/common/ModuleStepsContainer/ModuleStepsContainer.tsx

```diff
@@ -381,11 +381,12 @@ export default function ModuleStepsContainer({
 	};
 
 	const handleFacingIssuesClick = (): void => {
-		trackEvent('Onboarding V2: Facing Issues Sending Data to SigNoz', {
+		logEvent('Onboarding V2: Facing Issues Sending Data to SigNoz', {
```

Comment:
NIT: we do not need any error / success handling here ? as in we are not concerned with API returns whatever response here ?

Reply:
yes, here we are not concerned about the response and also dont want to block the process due to this call.

Reply:
so if somebody wants to use the `logEvent` they will have to process the entire hook code. We should keep it separate.